src/Polytopes/PolyhedralFan/properties.jl

```diff
@@ -361,3 +361,185 @@ true
+    newindex = size(rays,1) + 1
+    newray = [sum(gens[:,i]) for i in 1 : size(gens,2)]
+    
+    # add this ray to form list of the new rays
+    newrays = zeros(Int64,length(rays)+1,length(rays[1]))
+    for i in 1 : length(rays)
+        for j in 1:length(rays[i])
+            newrays[i,j] = rays[i][j]
+        end
+    end
+    for j in 1:length(newray)
+        newrays[length(rays)+1,j] = newray[j]
+    end
+    
+    # identify all maximal cones in the new fan
+    indices = [i for i in maxcones[n]]
+    push!(indices, newindex)
+    d = dim(cone)
+    test_cones = Oscar.Hecke.subsets(indices,d)
+    newmaxcones = Vector{Int}[];
+    for i in 1 : length(maxcones)
+        if i != n
+            push!(newmaxcones,maxcones[i])
+        end
+    end
+    for i in 1 : length(test_cones)
+        if test_cones[i] != maxcones[n]
+            push!(newmaxcones, [Int(k) for k in test_cones[i]])
+        end
+    end
+    newmaxcones = IncidenceMatrix(newmaxcones)
+    
+    # return the new fan
+    return PolyhedralFan(newrays, newmaxcones)
+    
+end
+
+###############################################################################
+## Cartesian/Direct product
+###############################################################################
+
+@doc Markdown.doc"""
+    *(PF1::PolyhedralFan, PF2::PolyhedralFan)
+
+Computes the Cartesian/direct product of two polyhedral fans.
+
+# Examples
+```jldoctest
+julia> normal_fan(Oscar.simplex(2))*normal_fan(Oscar.simplex(3))
+A polyhedral fan in ambient dimension 5
+```
+"""
+function Base.:*(PF1::PolyhedralFan, PF2::PolyhedralFan)
+    # extract dimensions of the fans
+    d1 = dim(PF1)
+    d2 = dim(PF2)
+    
+    # construct rays of the new fan
+    rays1 = [[Int(i) for i in r] for r in eachrow(Oscar.Polymake.common.primitive(pm_fan(PF1).RAYS))]
+    rays2 = [[Int(i) for i in r] for r in eachrow(Oscar.Polymake.common.primitive(pm_fan(PF2).RAYS))]
+    newrays = zeros(Int64,length(rays1)+length(rays2),d1+d2)
+    for i in 1 : length(rays1)
+        for j in 1:length(rays1[i])
+            newrays[i,j] = rays1[i][j]
+        end
+    end
+    for i in 1 : length(rays2)
+        for j in 1:length(rays2[i])
+            newrays[i+length(rays1),j+d1] = rays2[i][j]
+        end
+    end
+
+    # construct max cones of the new fan
+    maxcones = Vector{Int}[];
+    maxcones1 = [findall(x->x!=0, l) for l in eachrow(pm_fan(PF1).MAXIMAL_CONES)]
+    maxcones2 = [findall(x->x!=0, l) for l in eachrow(pm_fan(PF2).MAXIMAL_CONES)]
+    maxcones2 = [[k +  pm_fan(PF1).N_MAXIMAL_CONES for k in c] for c in maxcones2]
+    for i in 1 : length(maxcones1)
+        for j in 1 : length(maxcones2)
+            newcone = [c for c in maxcones1[i]]
+            for k in 1 : length(maxcones2[j])
+                push!(newcone, maxcones2[j][k])
+            end
+            push!(maxcones,newcone)
+        end
+    end
+    
+    # return the new fan
+    return PolyhedralFan(newrays, IncidenceMatrix(maxcones))
+    
```

Comment:
```suggestion
    return PolyhedralFan(Polymake.fan.product(pm_fan(PF1), pm_fan(PF2)))
```
@benlorenz just showed me, that "cartesian" is not in the name of the function in polymake. But maybe this just works? We think it is the same.

Reply:
If this is already in polymake, then by all means let us use it.